docs/src/_guide/orchestration.md

```diff
@@ -13,34 +13,37 @@ When a new pipeline schedule is created using the [UI](/reference/ui) or [CLI](/
 
 ## Create a Schedule
 
-### Specifying extractors and loaders
+### Scheduling predefined jobs
 
-To regularly schedule your ELT to run, use the ["Pipelines" interface in the UI](/reference/ui#pipelines), or the [`meltano schedule` command](/reference/command-line-interface#schedule):
+To regularly schedule your pipeline to run first define it as a [job](/reference/command-line-interface#job) within your project.
+Then you can schedule it directly using the `meltano schedule add` command:
 
 ```bash
-meltano schedule add [SCHEDULE_NAME] --extractor [EXTRACTOR_NAME] --loader [TARGET_NAME] --interval [INTERVAL]
-```
-
-Example:
+# Define a job
+meltano job add tap-gitlab-to-target-postgres-with-dbt --tasks "tap-gitlab target-postgres dbt:run"
 
-```bash
-meltano schedule add carbon__sqlite --extractor tap-carbon-intensity --loader target-sqlite --interval="@daily"
+# Schedule the job
+meltano schedule add daily-gitlab-load --job tap-gitlab-to-target-postgres-with-dbt --interval '@daily'
 ```
 
 Now that you've scheduled your first pipeline, you can load the "Pipeline" page in the UI and see it show up.
 
-### Scheduling predefined jobs
+### Specifying extractors and loaders
 
-If you've defined a [job](/reference/command-line-interface#job) within your project, you can schedule it directly without having to manually pass its plugins to the `meltano schedule add` command:
+If you'd prefer not to define a job and just schedule a simple ELT pipeline instead you can pass an Extractor, Loader, and Transform directly via the CLI:
```

Comment:
During the office hours I've heard us steer folks explicitly away from scheduling EL(T) tasks in 2.0. Should we maybe not directly mention the T portion here?

Is it worth calling out that this using `meltano elt` under the hood vs `meltano run`.  New users might not care, but more experienced users might?

/cc @aaronsteers @DouweM @tayloramurphy

Reply:
It's also occurring to me that we could maybe just drop this. Since this is more of a "How to Guide" rather than a comprehensive reference doc, I think it would be reasonable to only walk through the recommended approaches.

Reply:
That's _**a really good point**_ actually. We probably could and even *should* be opinionated here right ?

Reply:
I went ahead and removed most references to UI and old-style EL(T) pipelines from this guide.  The `meltano.yml` and CLI reference docs both still mention the old-style EL(T) approach to running, so that material is still there for users that need to reference it. But this guide now explicitly recommends the job approach since it's the recommended and better supported approach.

CC @DouweM @aaronsteers @tayloramurphy